imu_ekf:
  ros__parameters:
    window_size: 20
    accel_threshold: 0.1
    gyro_threshold: 0.01
    bias_random_walk: 1e-7
    zupt_noise: 0.001
    process_noise_position: 0.01
    process_noise_velocity: 0.05
    process_noise_orientation: 0.001
    process_noise_bias: 1e-6
    init_samples: 100
    imu_axis_remap: 'yxz'
    imu_axis_signs: '+-+'
